tests/tcp.rs

```diff
@@ -203,55 +192,52 @@ fn merge_and_fork() {
         sinks::socket::SocketSinkConfig::make_basic_tcp_config(out_addr2.to_string()),
     );
 
-    let mut rt = Runtime::with_thread_count(2).unwrap();
```

Comment:
This one looks like it requires `core_threads = 2`

Reply:
Currently, CI is running but if I remember correctly this test is fine with 1 thread.

Reply:
The change to use two threads originated from here: https://github.com/timberio/vector/pull/3340
It seems you have enough context on what's going on, so this should be fine if you say so :smile: